lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp

```diff
@@ -270,6 +270,48 @@ class ReductionOpConversionPattern : public OpConversionPattern<TTIROpTy> {
   }
 };
 
+class BroadcastOpConversionPattern
+    : public OpConversionPattern<ttir::BroadcastOp> {
+  using OpConversionPattern<ttir::BroadcastOp>::OpConversionPattern;
+
+public:
+  LogicalResult
+  matchAndRewrite(ttir::BroadcastOp op, ttir::BroadcastOp::Adaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+
+    // Fold the BroadcastOp only if it is Operand 0 of the instruction
+    // TODO(uazizTT): Remove this restriction for operand number once implicit
```

Comment:
Please file an issue and link to comment.

Reply:
Linked to issue #1591